README.md

```diff
@@ -1,2 +1,55 @@
-[![Join the chat at https://gitter.im/open-telemetry/opentelemetry-ruby](https://badges.gitter.im/open-telemetry/opentelemetry-ruby.svg)](https://gitter.im/open-telemetry/opentelemetry-ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![Build Status](https://travis-ci.org/open-telemetry/opentelemetry-ruby.svg?branch=master)](https://travis-ci.org/open-telemetry/opentelemetry-ruby)
+# OpenTelemetry Ruby
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![Travis CI][ci-image]][ci-url]
+[![Apache License][license-image]][license-image]
+
+The Ruby [OpenTelemetry](https://opentelemetry.io/) client.
+
+## Contributing
+
+We'd love your help! Use tags [good first issue][issues-good-first-issue] and
+[help wanted][issues-help-wanted] to get started with the project.
+
+The Ruby special interest group (SIG) meets regularly. See the OpenTelemetry
+[community page][ruby-sig] repo for information on this and other language SIGs.
+
+
+## Release Schedule
+
+OpenTelemetry Ruby is under active development. Our goal is to release an
+_alpha_ version of the library at the end of September 2019. This release isn't
+guaranteed to conform to a specific version of the specification, and future
+releases will not attempt to maintain backwards compatibility with the alpha
+release.
+
+| Component                   | Version | Target Date       |
+| --------------------------- | ------- | ----------------- |
+| Tracing API                 | Alpha   | Unknown           |
+| Tracing SDK                 | Alpha   | Unknown           |
+| Metrics API                 | Alpha   | Unknown           |
+| Metrics SDK                 | Alpha   | Unknown           |
+| Jaeger Trace Exporter       | Alpha   | Unknown           |
+| Zipkin Trace Exporter       | Alpha   | Unknown           |
```

Comment:
Zipkin exporter isn't a requirement for Alpha, I think?

Reply:
@fbogsany It's not technically part of the alpha requirements - seems that folks are putting that in because they already have it done. I've removed it from our schedule.